pom.xml

```diff
@@ -3937,6 +3937,10 @@
                 <groupId>org.slf4j</groupId>
                 <artifactId>slf4j-reload4j</artifactId>
               </exclusion>
+              <exclusion>
+                <groupId>org.bouncycastle</groupId>
+                <artifactId>bcprov-jdk15on</artifactId>
```

Comment:
So there isn't actually a runtime dependency on this jar?

Reply:
It is introduced by HDFS-15098, for adding SM4 encryption support. We need further tests to see if we need to introduce the dependency manually in HBase. For now we only depend on bouncycastle related jars in test scope in HBase.

For now I think just exclude it is fine.

Reply:
Tried locally, seems there are still problem while actually running tests against 3.4.0.

Anyway, let's get this in to at least make compilation pass for 3.4.0, and then open another issue for addressing the test issues.